docs/code-of-conduct.rst

```diff
@@ -57,36 +75,87 @@ Sponsors, affiliates, and exhibitors
 
 When you sponsor a Write the Docs event, we welcome you as a member of our community, and we expect you to be respectful to the community you operate within.
 
-All exhibitors in the expo hall, sponsor or vendor booths, or similar activities are also subject to the code of conduct. In particular, exhibitors should not use sexualized images, activities, or other material. Booth staff (including volunteers) must not use sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.
+All exhibitors in the expo hall, sponsor or vendor booths, or similar activities are also subject to the Code of Conduct. In particular, exhibitors should not use sexualized images, activities, or other material. Booth staff (including volunteers) must not use sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.
 
 In addition, sponsors and affiliates of conference, meetups, and online activities should not employ aggressive recruiting techniques, invasive marketing behavior, or similar actions towards community members. In case of violations, sponsors might be sanctioned and expelled from the event or activity with no return of the sponsorship contribution.
 
-What to do in case of violations
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+What can happen if the CoC is violated?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In case of a Code of Conduct violation, some of the most common actions organisers may take are:
+
+* No action (if the team determines no violation occurred).
+* A private or pubic reprimand.
+* Demanding that a public apology is made.
+* Demanding that a participant stops their behaviour.
+* Demanding that a participant prevents further contact with certain other participants.
+* Not publishing the video of a conference talk.
+* Cancelling a conference talk.
+* Removing a participant from the conference, meetup or online space, without refund.
+
+The action taken is at the discretion of the Code of Conduct team. Participants are expected to comply immediately, and further action may be taken in case a participant does not comply. A record will be kept of all incidents.
+
+What to do in case of violations?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If a Code of Conduct incident happens that affects you, or witness it affecting someone else, please contact the appropriate CoC team immediately. See the contact details below for the different teams for different spaces. In all cases, you may email conduct@writethedocs.org, but contacting local organisers may be faster.
+
+*TODO: who is behind conduct@wtd?*
```

Comment:
Core team at the moment, but maybe we can add the Slack moderators when we have them formalized

Reply:
I would not recommend that. That group is way too large. Perhaps there should be a separate address for that, as a first reporting point for Slack.